cmake/dependencies.cmake

```diff
@@ -15,9 +15,7 @@ if(THREADING STREQUAL "OMP")
     find_package(OpenMP)
 
     if(NOT OpenMP_CXX_FOUND)
-        message(WARNING "Compiler does not support OpenMP standard. Falling back to SEQ threading")
-        set(THREADING "SEQ")
-        set(ENABLE_INTEL_OPENMP OFF)
```

Comment:
don't we need to recommend users to disable INTEL_OPENMP as possible solution too?